src/s3-tables-mcp-server/awslabs/s3_tables_mcp_server/utils.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+"""Common utilities and helpers for S3 Tables MCP Server."""
+
+import os
+from functools import wraps
+from typing import Optional
+
+import boto3
+from botocore.config import Config
+
+from awslabs.s3_tables_mcp_server.constants import MCP_SERVER_VERSION
```

Comment:
I believe with the change:
```suggestion
from awslabs.s3_tables_mcp_server import __version__
```